tests/RobotFramework/tests/tedge_agent/http_server/entity_store_api.robot

```diff
@@ -1,5 +1,6 @@
 *** Settings ***
 Resource            ../../../resources/common.resource
+Library             Collections
```

Comment:
Is this line actually needed? I ran the test without it and it passed.

Reply:
Resolved by f837935c